tools/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+    "extends": "../tsconfig.json",
+    "compilerOptions": {
+      "rootDir": "..",
+      "outDir": "../build"
```

Comment:
curious why we can't use the top level tsconfig?

Reply:
Since we decided to checking the proto JSON we don't need to have this anymore. @LibanOdowa is removing it

Reply:
Done.